start.py

```diff
@@ -55,7 +56,17 @@ def main(simulation_parameters):
             allagents.do('filobl')
             insurancecustomers.do('check_risk')
 
-        simulation.graphs()
+        if not isTravis:
+            simulation.graphs()
```

Comment:
(e.g. https://github.com/EconomicSL/auctions-simulation-example/blob/master/periodic-double-auction-analytics.ipynb) Also, Jupyter notebooks, if versioned, can be cited from its hash while web server graphs can't. The `@gui` will have its purpose for interactive exploration.